





<!DOCTYPE html>
<html lang="en">
<head>
  <link rel="stylesheet" href="../../static/bootstrap-3.1.1/css/bootstrap.min.css">
  <link rel="stylesheet" href="../../static/css/richard.css">
  <link rel="stylesheet" href="../../static/browserid/persona-buttons.css" />
  <link rel="search" type="application/opensearchdescription+xml" title="pyvideo.org search" href="../../search/xml">
  <title>pyvideo.org - Exploring disease genetics from thousands of individual genomes with Gemini; Sci...</title>
  <link rel="shortcut icon" type="image/x-icon" href="http://pyvideo.org/favicon.ico">

  
    <meta name="keywords" content="Tech">
  
    <meta name="description" content="Authors: Quinlan, Aaron, University of Virginia; Paila, Uma, University of Virginia; Chapman, Brad, Harvard School of Public Health; Kirchner, Rory,

Track: Bioinformatics

The throughput of DNA sequencing has increased by five orders of magnitude in the last decade and geneticists can now sequence a complete human genome in 24 hours for less than $5000. This tremendous increase in efficiency has led to large-scale studies of the relationship between inherited genetic variation and human disease. While collecting genetic variation from the genomes of thousands of humans is now possible, unraveling the genetic basis of disease remains a tremendous analytical challenge. Interpretation is especially difficult since many genetic variants associated with human disease lie outside the genomic regions that encode genes. To address this challenge, we have developed GEMINI, a flexible Python analysis framework for exploring human genetic variation. By leveraging Numpy, SQLITE, and several powerful Python packages in the genomics domain, GEMINI integrates genome-scale genetic variation from 1000s of individuals with a wealth of genome annotations that are crucial for disease interpretation.

GEMINI provides a powerful analysis framework allowing researchers to conduct otherwise complicated analyses with an easy to use analysis interface. It provides methods for ad hoc data exploration, a programming interface for custom analyses, and both command line and graphical tools for common analysis tasks. We demonstrate GEMINI&#39;s utility for exploring variation for personal genomes and family based genetic studies. Thanks to advances such as IPython.parallel, we further illustrate the framework&#39;s ability to scale to studies involving thousands of human samples.">
  

  
</head>
<body>
  <input type='hidden' name='csrfmiddlewaretoken' value='DEhrCXO8g33mPB1YNZ7JRQq9NQMV9Bvc' />
  
<div id="browserid-info"
     style="display: none;"
     data-info="{&quot;loginUrl&quot;: &quot;/browserid/login/&quot;, &quot;requestArgs&quot;: {}, &quot;logoutUrl&quot;: &quot;/browserid/logout/&quot;, &quot;csrfUrl&quot;: &quot;/browserid/csrf/&quot;}">
</div>

  <a name="top"></a>
  <nav class="navbar navbar-default" role="navigation">
    <div class="container-fluid">
      <div class="navbar-header">
        <a class="navbar-brand" href="../../index.html">pyvideo.org</a>
      </div>
      <div class="collapse navbar-collapse">
        <ul class="nav navbar-nav">
          <li class="dropdown">
            <a href="exploring-disease-genetics-from-thousands-of-indi-.html#" class="dropdown-toggle" data-toggle="dropdown">
              Browse <b class="caret"></b>
            </a>
            <ul class="dropdown-menu">
              <li><a href="../../category.html">Conferences/user groups</a></li>
              <li><a href="../../speaker/index.html">Speakers</a></li>
            </ul>
          </li>
        </ul>
        <form class="navbar-form navbar-left" role="search" action="../../search.1.html">
          <div class="form-group">
            <input type="hidden" name="models" value="videos.video">
            <input type="text" class="form-control" name="q" placeholder="Search" autofocus value="">
          </div>
        </form>
        

        <ul class="nav navbar-nav navbar-right">
          
            <li><div class="signin">

<a href="exploring-disease-genetics-from-thousands-of-indi-.html#" data-next="/" class="browserid-login persona-button" >
	<span>Sign in</span>
</a>
</div></li>
          
        </ul>
      </div>
    </div>
  </nav>

  <div id="content" class="container">
    
      <div class="row">
        <div class="col-md-12">
          
            <div class="alert alert-info">
              <i class="icon-info-sign"></i> <strong>Sorry!</strong>
              <p>pyvideo is closing down at some point in 2016. <a href="http://bluesock.org/~willkg/blog/pyvideo/status_20160115.html">More details</a></p>
            </div>
          
        </div>
      </div>
    

    


<div class="row section" vocab="http://schema.org" typeof="VideoObject">
  <div id="videobox" class="col-md-12">
    
    <div class="clearfix">
      <div class="videobox-inner">
        
          
            <div class="amara-embed" data-height="480px" data-width="854px" data-url="http://www.youtube.com/watch?v=p-UWmDG6yj4"></div>
          
        
      </div>
    </div>
  </div>
</div>

<div class="row">
  <div id="videobox" class="col-md-8">
    <div class="section">
      <h3 class="centered" property="name">Exploring disease genetics from thousands of individual genomes with Gemini; SciPy 2013 Presentation</h3>
      <link property="thumbnailUrl" href="http://i1.ytimg.com/vi/p-UWmDG6yj4/hqdefault.jpg"/>
      <link property="sameAs" href="exploring-disease-genetics-from-thousands-of-indi-.html"/>
    </div>

    

    
      <div class="section">
        <h4>Summary</h4>
        <div class="video-details-summary" property="description"><p>Authors: Quinlan, Aaron, University of Virginia; Paila, Uma, University of Virginia; Chapman, Brad, Harvard School of Public Health; Kirchner, Rory,</p>
<p>Track: Bioinformatics</p>
<p>The throughput of DNA sequencing has increased by five orders of magnitude in the last decade and geneticists can now sequence a complete human genome in 24 hours for less than $5000. This tremendous increase in efficiency has led to large-scale studies of the relationship between inherited genetic variation and human disease. While collecting genetic variation from the genomes of thousands of humans is now possible, unraveling the genetic basis of disease remains a tremendous analytical challenge. Interpretation is especially difficult since many genetic variants associated with human disease lie outside the genomic regions that encode genes. To address this challenge, we have developed GEMINI, a flexible Python analysis framework for exploring human genetic variation. By leveraging Numpy, SQLITE, and several powerful Python packages in the genomics domain, GEMINI integrates genome-scale genetic variation from 1000s of individuals with a wealth of genome annotations that are crucial for disease interpretation.</p>
<p>GEMINI provides a powerful analysis framework allowing researchers to conduct otherwise complicated analyses with an easy to use analysis interface. It provides methods for ad hoc data exploration, a programming interface for custom analyses, and both command line and graphical tools for common analysis tasks. We demonstrate GEMINI's utility for exploring variation for personal genomes and family based genetic studies. Thanks to advances such as IPython.parallel, we further illustrate the framework's ability to scale to studies involving thousands of human samples.</p></div>
        <meta property="caption" content="Authors: Quinlan, Aaron, University of Virginia; Paila, Uma, University of Virginia; Chapman, Brad, Harvard School of Public Health; Kirchner, Rory,
Track: Bioinformatics
The throughput of DNA sequencing has increased by five orders of magnitude in the last decade and geneticists can now sequence a complete human genome in 24 hours for less than $5000. This tremendous increase in efficiency has led to large-scale studies of the relationship between inherited genetic variation and human disease. While collecting genetic variation from the genomes of thousands of humans is now possible, unraveling the genetic basis of disease remains a tremendous analytical challenge. Interpretation is especially difficult since many genetic variants associated with human disease lie outside the genomic regions that encode genes. To address this challenge, we have developed GEMINI, a flexible Python analysis framework for exploring human genetic variation. By leveraging Numpy, SQLITE, and several powerful Python packages in the genomics domain, GEMINI integrates genome-scale genetic variation from 1000s of individuals with a wealth of genome annotations that are crucial for disease interpretation.
GEMINI provides a powerful analysis framework allowing researchers to conduct otherwise complicated analyses with an easy to use analysis interface. It provides methods for ad hoc data exploration, a programming interface for custom analyses, and both command line and graphical tools for common analysis tasks. We demonstrate GEMINI&#39;s utility for exploring variation for personal genomes and family based genetic studies. Thanks to advances such as IPython.parallel, we further illustrate the framework&#39;s ability to scale to studies involving thousands of human samples."/>
      </div>
    

    

    
  </div>

  <div class="col-md-4">
    <div id="sidebar" class="well">
      

      <dl>
        
        <dt>Category</dt>
        <dd>
          <a href="../../category/37/scipy-2013.html">SciPy 2013</a>
        </dd>

        <dt>Speakers</dt>
        <dd>
          
            Unknown
          
        </dd>

        
          <dt>Language</dt>
          <dd>English
            <meta property="inLanguage" content="en"/>
          </dd>
        

        

        
          <dt>Recorded</dt>
          <dd>July 1, 2013
            <meta property="dateCreated" content="2013-07-01"/>
          </dd>
        

        
          <dt>Last updated</dt>
          <dd>April 8, 2014
            <meta property="dateModified" content="2014-04-08T20:28:26.394627"/>
            <meta property="datePublished" content="2014-04-08T20:28:26.394627"/>
          </dd>
        

        <dt>Video origin</dt>
        <dd>
          
            <a href="http://www.youtube.com/watch?v=p-UWmDG6yj4" property="embedUrl">http://www.youtube.com/watch?v=p-UWmDG6yj4</a>
          
        </dd>

        <dt>Download</dt>
        <dd>
          
            No downloadable files.
          
        </dd>

        
          <dt>Metadata</dt>
          <dd>
            <a href="../../api/v2/video/1998">JSON</a>
          </dd>
        

        <dt>Copyright/License Information</dt>
        <dd>
          
            http://www.youtube.com/t/terms
          
        </dd>

        <dt>Tags</dt>
        <dd>
          
            <span property="keywords">Tech</span>
          
          <meta property="keywords" content="SciPy 2013"/>
        </dd>
      </dl>
    </div>
  </div>
</div>


    
    <div id="footer">
      <a name="footer"></a>
      <div class="row">
        <div class="col-md-6">
          <p>pyvideo.org</p>
          <p>
            Run by Sheila Miguez and Will Kahn-Greene Questions,
            comments or concerns? Send an email to Will at willkg at
            bluesock dot org.</p>
          <p>
            Powered by <a href="https://github.com/pyvideo/richard">richard</a>.
          </p>
          <p>
            Hosting for the project is graciously provided by
            <a href="http://www.rackspace.com/cloud">Rackspace</a>.
          </p>
        </div>
        <div class="col-md-4">
          <ul class="unstyled">
            <li><a href="../../pages/about.html">About this site</a></li>
            <li><a href="../../stats/index.html">Site stats</a></li>
            <li><a href="http://github.com/pyvideo/pyvideo/issues/new">Report an issue</a></li>
            <li><a href="http://github.com/pyvideo/pyvideo/issues/new">Suggest a conference to add</a></li>
          </ul>
        </div>
        <div class="col-md-2">
          <a class="pull-right" href="exploring-disease-genetics-from-thousands-of-indi-.html#top">Back to top</a>
        </div>
      </div>
    </div>
    

  </div>

  


  
  <script type="text/javascript" src="../../static/jquery-1.11.0/jquery-1.11.0.min.js"></script>
  <script type="text/javascript" src="../../static/bootstrap-3.1.1/js/bootstrap.min.js"></script>
  <script type="text/javascript" src="https://login.persona.org/include.js"></script>
<script type="text/javascript" src="../../static/browserid/api.js"></script>
<script type="text/javascript" src="../../static/browserid/browserid.js"></script>
  <script type="text/javascript">
    var _gaq = _gaq || [];
    _gaq.push(['_setAccount', 'UA-29832059-1']);
    _gaq.push(['_trackPageview']);

    (function() {
      var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
      ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
    })();
  </script>


  <script type="text/javascript" src="http://amara.org/embedder-iframe"></script>
  <script type="text/javascript" src="../../static/videos/js/videos.js"></script>
  <script type="text/javascript">
    $(document).ready(function() {
      // react to changes after page load
      $(window).on('hashchange', function() {
        updateVideoOffset('unisubs');
      });

      updateVideoOffset('unisubs');
    });
  </script>

</body>
</html>
